src/colorpickr.test.tsx

```diff
@@ -156,6 +160,13 @@ describe('Colorpickr', () => {
       expect(element.style.backgroundColor).toEqual('rgb(255, 0, 0)');
     });
 
+    test('emitOnChange: false does not invoke onChange handler', () => {
+      mockedInstance.overrideValue('red', true, false);
```

Comment:
This one took me a while to understand, I really prefer having something like

```suggestion
      mockedInstance.overrideValue('red', { emitOnChange: false });
```

instead, to document what you're passing without activating IntelliSense and also so that you don't have to specify optional parameters just because of order.

It's definitely not a blocker, but nameless Boolean parameters always give me pause

Reply:
Yeah, I agree and it's a good change to make... I was mostly following the convention that's unfortunately already in place for  [`shouldUpdateInitialValue`](https://github.com/mapbox/react-colorpickr/blob/fc76cb06da2765fff0d37c77c835c741d1168909/src/colorpickr.tsx#L118) which is a passed parameter and not an object of named options.